src/lib/pure/ketrew_edsl.mli

```diff
@@ -213,22 +213,23 @@ val file_target:
 
 val daemonize :
   ?starting_timeout:float ->
+  ?call_script:(string -> string list) ->
   ?using:[`Nohup_setsid | `Python_daemon] ->
   ?host:Host.t ->
   Program.t ->
   Ketrew_target.Build_process.t
-(** Create a “daemonize” build process. *)
+(** Create a “daemonize” build process:
 
-(*
-val direct_execution :
-  ?host:Host.t -> Program.t -> Ketrew_target.Build_process.t
-(** Create a direct process (not “long-running”). *)
+    - [?host]: the [Host.t] on which the program is to be run.
+    - [?starting_timeout]: how long to wait before considering that a
+      script failed to start (default: [5.] seconds).
+    - [?call_script]: how to call the generated shell script
```

Comment:
What about instead of 'how to call the generated shell script' , the sequence of terminal commands to execute the script'?

I think that something specific about what you're invoking would be helpful.